app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::UsersController < ApplicationController
+
+  def show 
+    user = User.includes(:schedule).find(params[:id])
+    puts user.schedule.inspect
```

Comment:
Probably want to remove the puts, but consider adding actual logging with the `Rails.logger`